packages/engine/src/scene/components/LinkComponent.ts

```diff
@@ -25,6 +25,12 @@ Ethereal Engine. All Rights Reserved.
 
 import { useEffect } from 'react'
 
+import { RouterState } from '@etherealengine/client-core/src/common/services/RouterService'
```

Comment:
the engine package cannot import from client-core. the link component should either dispatch an action that is consumed in a client system, or something similar to detatch this from the core engine